src/core/render/index.js

```diff
@@ -132,8 +132,13 @@ export function renderMixin(proto) {
 
     if (hideSidebar) {
       // FIXME : better styling solution
```

Comment:
LGTM

lets remove this comment for now

Reply:
done.